doc/testing.md

```diff
@@ -135,10 +135,19 @@ the first N tiers they can afford to run, but at least tier1.
 
 A brief description of the tiered test groups:
 
-- `tier1`: This is the lowest test tier. Multiple developers run these tests
+- `tier1`: This is the most fundamental test tier.
+  Roughly speaking, a failure of a test in this tier has the potential to
+  indicate a problem that would affect many Java programs. Tests in `tier1` include
+  tests of HotSpot, libraries in the `java.base` module, and the `javac` compiler.
```

Comment:
Not all libraries in `java.base` are tested in tier1. For instance, `java.net` or `java.nio` are part of tier2.
My worry here is that developers may misinterpret this statement and think the code they have modified in `java.base` will be tested if they run `tier1`. Maybe this sentence could be slightly altered to say that:

> ... `tier1` include tests of HotSpot, **some** libraries in the `java.base` module, ...

Reply:
Will changed to "selected libraries in java.base"; thanks.